index.js

```diff
@@ -8,6 +8,10 @@ module.exports = function() {};
 module.exports.pitch = function(remainingRequest) {
     this.cacheable && this.cacheable();
     var query = loaderUtils.parseQuery(this.query);
+    var chunkName = loaderUtils.interpolateName(this, 'routes/' + query.name, {
```

Comment:
I'm thinking maybe automatically putting the chunks in a subfolder _by default_ is a bad idea, would maybe be better to just leave behavior unchanged but mention how to do that in the README?